WireUI/.swiftpm/WireReusableUIComponents.xctestplan

```diff
@@ -9,6 +9,11 @@
     }
   ],
   "defaultOptions" : {
+    "commandLineArgumentEntries" : [
+      {
+        "argument" : "-NSShowNonLocalizedStrings YES"
```

Comment:
Out of interest what does this argument do. Highlight strings that are not localized in `en`?

Reply:
It capitalises all letters.